crates/ruma-common/src/events/room/message/content_serde.rs

```diff
@@ -49,3 +49,82 @@ impl<'de> Deserialize<'de> for MessageType {
         })
     }
 }
+
+#[cfg(feature = "unstable-msc3488")]
+pub(in super::super) mod msc3488 {
```

Comment:
Might as well use `pub(crate)`?

Reply:
If you want bu currently this is equivalent to `crate::events::room::message`

Reply:
It just looks funny to me. But can leave as-is.